server/modules/activityLog/activityLog.js

```diff
@@ -1,10 +1,6 @@
-const db = require('@server/db/db')
+import * as R from 'ramda'
 
```

Comment:
I would move this to '@common/activityLog/activityLog.js'

Reply:
it's used only by the server modules now, I'd move it only if needed.

Reply:
it will be needed in the UI. i would move it in this pull request, otherwise in the next one we'll need to review the same files again